### PR TITLE
Fixed lang term when there is no evidence

### PR DESCRIPTION
**Pull request by mpharoah-d2l**

### CODE REVIEW

File: build/lang/ar.js [hunk 1]
```diff
@@ -21,6 +21,6 @@ D2L.PolymerBehaviors.OutcomesUserProgress.LocalizeBehavior.LangArBehavior = {
     "trend": "Trend",
     "evidence": "Evidence",
     "close": "إغلاق",
-    "noItemsFound": "لم يتم العثور على أي مواد."
+    "noEvidence": "This {outcome, select, competencies {competency} expectations {expectation} objectives {objective} outcomes {outcome} standards {standard} other {standard}} has not yet been assessed"
```
Comment (mpharoah-d2l) [inline build/lang/ar.js]: What this does is select the appropriate translation based off of the provided enum value (competencies/expectations/objectives/outcomes/standards)

Comment (mpharoah-d2l) [reply build/lang/ar.js]: So we would pass one of these values into the `outcomes-term` attribute, then our lang terms can select the appropriate translation based on that value

File: src/outcomes-user-progress/outcomes-user-progress.js [hunk 2]
```diff
@@ -24,6 +24,7 @@ export class OutcomesUserProgress extends mixinBehaviors(
                 hidden$="[[!_showDetails]]"
                 href="[[_detailsHref]]"
                 token="[[token]]"
+				outcome-term="[[outcomeTerm]]"
```
Comment (mpharoah-d2l) [inline src/outcomes-user-progress/outcomes-user-progress.js]: Tabs and Spaces: Civil War

File: src/outcomes-user-progress/outcomes-user-progress.js [hunk 3]
```diff
@@ -24,7 +24,7 @@ export class OutcomesUserProgress extends mixinBehaviors(
                 hidden$="[[!_showDetails]]"
                 href="[[_detailsHref]]"
                 token="[[token]]"
-				outcome-term="[[outcomeTerm]]"
+                outcome-term="[[outcomeTerm]]"
```
Comment (rgoessl) [inline src/outcomes-user-progress/outcomes-user-progress.js]: How did this pass lint?

**Pull request merged**